### ISSUE TITLE
PLAYBACK_TRACK_CHANGED event is missing parameters

### ISSUE THREAD

**Issue by dcvz** [doublesymmetry/react-native-track-player#1389]
The `PLAYBACK_TRACK_CHANGED` event used to have the following payload:

```java
Bundle bundle = new Bundle();
if (prevIndex != null) bundle.putInt("track", prevIndex);
bundle.putDouble("position", Utils.toSeconds(prevPos));
if (nextIndex != null) bundle.putInt("nextTrack", nextIndex);
service.emit(MusicEvents.PLAYBACK_TRACK_CHANGED, bundle);
```

at the moment we're sending this:
```kotlin
putDouble(POSITION_KEY, 0.0)
putInt(NEXT_TRACK_KEY, player.currentIndex)
```

we're hardcoding the position to `0` instead of the ending position of the last track and omitting the previous track index.

**Comment by jspizziri** [comment]
@dcvz part of this is fixed by #1501 , but it looks like the getting the last position of the previous track is going to require changes to KotlinAudio. Attempted to sort that out bug got hung up.

**Comment by jspizziri** [comment]
fixed via https://github.com/doublesymmetry/react-native-track-player/pull/1501 and split out into https://github.com/doublesymmetry/react-native-track-player/issues/1613
